assets/src/edit-story/elements/image.js

```diff
@@ -16,13 +17,19 @@ const Element = styled.img`
 	${ ElementWithRotation }
 `;
 
-function Image( { src, width, height, x, y, rotationAngle } ) {
+function Image( { src, width, height, x, y, rotationAngle, isBackground } ) {
+	const {
+		actions: {
+			percentToEditorPx,
+		},
+	} = useUnits();
 	const props = {
-		width,
-		height,
-		x,
-		y,
-		rotationAngle,
+		// TODO: what to do with 100% width/height?
+		width: isBackground ? percentToEditorPx( 100, 'w' ) : width,
+		height: isBackground ? percentToEditorPx( 100, 'v' ) : height,
```

Comment:
`'v'`? I know that any non-`'w'` value works, but isn't `h` the logical choice? 😃 

```suggestion
		height: isBackground ? percentToEditorPx( 100, 'h' ) : height,
```

Reply:
Right. Do we use any asserts? I couldn't find any yet.

Reply:
I think its unusual to use `assert` in React development. Also here we could use a bool rather than magic strings - `isHorizontal` perhaps. If not, then an enum would be better IMO.

Reply:
Ok. Switching to constants for now.

Reply:
But for the future: let's do enums.